go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.26
-	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.24
+	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.25
```

Comment:
does the fix in arwen is backwards compatible?

Reply:
According to the latest commit, it looks like it should not break the backwards compatibility.